keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuatorTests.kt

```diff
@@ -196,6 +201,42 @@ internal class ResourceActuatorTests : JUnit5Minutests {
               }
             }
 
+            context("when the a actuation launched event exists in resource history, check other events before publishing ResourceDeltaResolved") {
+              before {
+                resourceRepository.appendHistory(ResourceActuationLaunched(resource, plugin1.name, emptyList()))
+                runBlocking {
+                  subject.checkResource(resource)
+                }
+              }
+              test("ResourceDeltaResolved was not published since the task is still running") {
+                verify(exactly = 0) { publisher.publishEvent(ofType<ResourceDeltaResolved>()) }
+              }
+
+              context("the task was finished successfully") {
```

Comment:
This should be outdented to be at the same level as the context for "last event is actuation launched", right?

Reply:
discussed privately.